backend/src/controllers/pendingCourses/getPendingCourses.ts

```diff
@@ -8,8 +8,10 @@ import { getPendingCoursesProvider } from 'db/providers/pendingCoursesProvider';
 const getPendingCourses = async (req: Request, res: Response, next: TMiddlewareCall) => {
   const { id: userId } = res.locals;
   try {
-    const user: IUser = await getPendingCoursesProvider(userId);
-    res.json(user.pendingCourses);
+    if (typeof userId === 'string') {
```

Comment:
Any cases when it will be something except string?

Reply:
`userId as string` maybe?